File: app/src/main/java/com/michalbaran/incomeapp/LoginActivity.java
package com.michalbaran.incomeapp;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.Button;
import android.widget.EditText;

public class LoginActivity extends AppCompatActivity {

    EditText editLogin;
    EditText editPassword;
    Button btnLogin;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.login_activity);
        editLogin.findViewById(R.id.editLogin);
        editPassword.findViewById(R.id.editPassword);
    }
}

File: app/src/main/java/com/michalbaran/incomeapp/LoginActivity.kt
package com.michalbaran.incomeapp

import androidx.appcompat.app.AppCompatActivity

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.EditText
import kotlinx.android.synthetic.main.login_activity.*

class LoginActivity : AppCompatActivity() {



    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.login_activity)
        //editLogin = findViewById(R.id.editLogin)
        //editPassword = findViewById(R.id.editPassword)
        btnLogin.setOnClickListener {
            val intent = Intent(this@LoginActivity, MainActivity::class.java)
            startActivity(intent)
        }
    }

}
